### PR TITLE
iex: add page

### PR DESCRIPTION
**Pull request by Schoonology**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by Schoonology** [comment]
This was intended to address the `iex` component of https://github.com/tldr-pages/tldr/issues/2680. (The `elixir` half may take more thought, and will come later.)

File: pages/common/iex.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# iex
+
+> IEx is the interactive shell for Elixir.
+> More information: <https://hexdocs.pm/iex>.
+
+- Start an empty interactive session:
```
Comment (navarroaxel) [inline pages/common/iex.md:6]: is `empty` for command history? I prefer the example description without empty:
```suggestion
- Start an interactive session:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @Schoonology. Just 1 comment.

**Comment by Schoonology** [comment]
@navarroaxel Again, completely reasonable. 👍🏼

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Pull request merged**